.env-example

```diff
@@ -28,14 +31,22 @@ AI_CONFIG_PREFIX=!config
 # You can use our hosted Speech API (for free) or host your own Speech API
 SPEECH_API_URL=https://speech-service.verlekar.com
 
+# Whisper API URL
+# You can use Whisper API for voice transcription
+WHISPER_API_URL=https://transcribe.whisperapi.com
```

Comment:
Can we use the actual OpenAI Whisper API instead of whisperapi.com?

Reply:
Oh the Open AI one does not take ogg directly and not returning language
Nvm let me add openai transcriber as well